src/rezplugins/build_system/custom.py

```diff
@@ -207,6 +207,19 @@ def _add_build_actions(cls, executor, context, package, variant,
             install_path=install_path
         )
 
+    @classmethod
+    def _add_pre_build_commands(cls, executor, variant, build_type, 
```

Comment:
drop these `_add_pre_build_commands` methods and just call `add_pre_build_commands` directly.

It's different for `_add_build_actions` because those may or may not just comprise of the 'standard' build actions (it's feasible that a specific build system may want to set an env var for eg). However, in the case of `pre_build_commands`, there's nothing about that that is specific to a build system.